ecms-social-integration/src/main/java/org/exoplatform/wcm/ext/component/activity/listener/ActivityListener.java

```diff
@@ -116,7 +121,7 @@ private void shareActivityFilesToSpace(ActivityLifeCycleEvent activityLifeCycleE
                 }
               }
             } catch (RepositoryException e) {
-              LOG.warn("Error while sharing document to space. Node uuid: " + fileToShare.getId(), e);
+              LOG.error("Error while sharing document to space. Node uuid: " + nodeUuid, e);
```

Comment:
Is it safe to add here (for each iteration) ?
`            finally {
              sessionProvider.close();
            }
`
cc @ahamdi

Reply:
it throws an exception of Kind "sessionprovider already closed"

Reply:
Yes it is needed to close it otherwise we will have some JCR session  leaks.